README.md

```diff
@@ -36,6 +36,59 @@ The client keys used with Parse are no longer necessary with parse-server.  If y
 * restAPIKey
 * dotNetKey
 
+#### OAuth Support
+
+parse-server supports 3rd party authentication with
+
+* twitter
+* meetup
+* linkedin
+* google
+* instagram
+* facebook
+
+You don't need to explicitely to anything but for Twitter
```

Comment:
"You don't need to explicitly do"

But.. Is this true?  Doesn't FB auth require setting the app id too?  Maybe this could say:

"Configuration options for these 3rd-party modules is done with the `oauth` option passed to ParseServer:"